### ISSUE TITLE
Show error calls a breakpoint

### ISSUE THREAD

**Issue by valmontechno** [ScratchAddons/ScratchAddons#6825]
### Why this would be helpful

[discord](https://discord.com/channels/806602307750985799/1168613311311511685)

### My suggestion

Add an option that automatically pause the project when a show error block is called.

**Comment by DNin01** [comment]
You could just use the breakpoint block...?

**Comment by WorldLanguages** [comment]
Yeah, this would be a nice addition but I'm not sure if we would add it on its own. The debugger also lacks filtering, search and other useful features. I recall some recent feedback or github issue that mentioned they wanted to see logs of broadcasts, but not the "tick" broadcast which some projects trigger once per frame.

**Comment by valmontechno** [comment]
There is also a bug that do overflow icons in other language.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/108832011/dc7c595f-2860-4e69-90ca-8d5e7d5fdec3)

**Comment by WorldLanguages** [comment]
> There is also a bug that do overflow icons in other language.

Is there an issue about this? I think I've seen it before.

**Comment by valmontechno** [comment]
https://discord.com/channels/806602307750985799/1083808657290039356
But there is no corresponding issue.

**Comment by WorldLanguages** [comment]
@valmontechno Could you create an issue about that bug?

**Comment by WorldLanguages** [comment]
+1 from feedback on adding filtering

**Comment by valmontechno** [comment]
> [@valmontechno](https://github.com/valmontechno) Could you create an issue about that bug?

#8216
